telemetry/engagementProfileEvents.js

```diff
@@ -0,0 +1,116 @@
+export const profileEvents = {
+  // follow button
+  'profile.follow.unblock': {
+    engagement_type: 'general',
+  },
+  'profile.follow.unmute': {
+    engagement_type: 'general',
+  },
+  'profile.follow.unfollow': {
+    engagement_type: 'general',
+  },
+  'profile.follow.follow': {
+    engagement_type: 'general',
+  },
+  'profile.follow.withdraw-follow-request': {
+    engagement_type: 'general',
+  },
+  'profile.follow.follow-request': {
+    engagement_type: 'general',
+  },
+  // more button and more options dropdown
+  'profile.more.open': {
+    engagement_type: 'general',
+  },
+  'profile.more.open_in_original_site': {
+    engagement_type: 'general',
+  },
+  'profile.more.share_account': {
+    engagement_type: 'general',
+  },
+  'profile.more.mention': {
+    engagement_type: 'general',
+  },
+  'profile.more.direct_message': {
+    engagement_type: 'general',
+  },
+  'profile.more.show_boosts': {
+    engagement_type: 'general',
+  },
+  'profile.more.hide_boosts': {
+    engagement_type: 'general',
+  },
+  'profile.more.add_note': {
+    engagement_type: 'general',
+  },
+  'profile.more.remove_note': {
+    engagement_type: 'general',
+  },
+  'profile.more.mute': {
+    engagement_type: 'general',
+  },
+  'profile.more.unmute': {
+    engagement_type: 'general',
+  },
+  'profile.more.block': {
+    engagement_type: 'general',
+  },
+  'profile.more.unblock': {
+    engagement_type: 'general',
+  },
+  'profile.more.block_domain': {
+    engagement_type: 'general',
+  },
+  'profile.more.unblock_domain': {
+    engagement_type: 'general',
+  },
+  'profile.more.report': {
+    engagement_type: 'general',
+  },
+  'profile.more.goto_pinned': {
+    engagement_type: 'general',
+  },
+  'profile.more.goto_favorites': {
+    engagement_type: 'general',
+  },
+  'profile.more.goto_mutes': {
+    engagement_type: 'general',
+  },
+  'profile.more.goto_blocks': {
+    engagement_type: 'general',
+  },
+  'profile.more.goto_domain_blocks': {
+    engagement_type: 'general',
+  },
+  // notifications
+  'profile.notify_start': {
+    engagement_type: 'general',
+  },
+  'profile.notify_stop': {
+    engagement_type: 'general',
+  },
```

Comment:
How do you feel about `profile.notify.start` and `profile.notify.stop` instead of using underscores?

Reply:
I sneaked this change into #68